src/test/test-procfs-util.c

```diff
@@ -52,8 +52,9 @@ int main(int argc, char *argv[]) {
                 log_info("Reducing limit by one to %"PRIu64"…", v-1);
 
                 r = procfs_tasks_set_limit(v-1);
-                log_info_errno(r, "procfs_tasks_set_limit: %m");
-                assert_se(r >= 0 || ERRNO_IS_PRIVILEGE(r) || r == -EROFS);
+                if (IN_SET(r, -ENOENT, -EROFS) || ERRNO_IS_PRIVILEGE(r))
+                        return log_tests_skipped_errno(r, "can't set tasks limit");
```

Comment:
We don't have 317bb217d3f21312e58d9efdc6d3739f7895a08c in RHEL 8, so the easiest way to fix this would be:

```suggestion
                        return EXIT_TEST_SKIP;
```

Reply:
Or backporting the mentioned commit, if you're brave enough ;)